packages/wonder-blocks-pill/src/components/pill.tsx

```diff
@@ -149,32 +151,10 @@ const Pill = React.forwardRef(function Pill(
 
 const styles: Record<string, any> = {};
 
-const pillStyles = StyleSheet.create({
-    wrapper: {
-        display: "inline-flex",
-        width: "fit-content",
-    },
-    wrapperSmall: {
-        paddingLeft: tokens.spacing.xSmall_8,
-        paddingRight: tokens.spacing.xSmall_8,
-        borderRadius: tokens.spacing.xxSmall_6,
-        // Minimum tap area recommendation for a11y
-        height: tokens.spacing.large_24,
-    },
-    wrapperLarge: {
-        paddingLeft: tokens.spacing.small_12,
-        paddingRight: tokens.spacing.small_12,
-        paddingTop: tokens.spacing.xxSmall_6,
-        paddingBottom: tokens.spacing.xxSmall_6,
-        borderRadius: tokens.spacing.large_24,
-        height: tokens.spacing.xLarge_32,
-    },
-});
-
-const _generateColorStyles = (
+const _generateStyles = (
     clickable: boolean,
     kind: PillKind,
-    size: "small" | "large",
+    size: PillSize,
```

Comment:
**suggestion:** Based on my previous comment, this makes me realize that we don't need the `size` argument here.